README.md

```diff
@@ -148,7 +148,10 @@ If you're using the default tenant, the URL is `ws://realtime-dev.localhost:4000
 | TENANT_MAX_EVENTS_PER_SECOND   | string  | The default value of maximum events per second that each tenant can support, used when creating a tenant for the first time. Defaults to '100'.                                                                                                                                                                                 |
 | TENANT_MAX_JOINS_PER_SECOND    | string  | The default value of maximum channel joins per second that each tenant can support, used when creating a tenant for the first time. Defaults to '100'.                                                                                                                                                                          |
 | SEED_SELF_HOST                 | boolean | Seeds the system with default tenant                                                                                                                                                                                                                                                                                            |
-
+| RUN_SCHEDULED_AFTER_IN_MS      | number  | Tells system when to start scheduled tasks after boot                                                                                                                                                                                                                                                                           |
+| RUN_SCHEDULED                  | boolean | Do you want to scheduled tasks to run                                                                                                                                                                                                                                                                                           |
+| MAX_CHILDREN_SCHEDULED_CLEANUP | number  | Maximum number of concurrent tasks working on scheduled cleanup                                                                                                                                                                                                                                                                 |
+| SCHEDULED_CLEANUP_TASK_TIMEOUT | number  | Timeout for each async task for scheduled cleanup                                                                                                                                                                                                                                                                               |
```

Comment:
A bit of a nit, but if we have the application_name of `realtime_janitor` we should change these to `RUN_JANITOR` etc cuz people might be searching our repo for "janitor" after seeing this application_name.